api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -34,6 +34,50 @@ const (
 	PrivateDNSZoneModeNone string = "None"
 )
 
+// UpgradeChannel - For more information see setting the AKS cluster auto-upgrade channel [https://docs.microsoft.com/azure/aks/upgrade-cluster#set-auto-upgrade-channel].
+type UpgradeChannel string
+
+const (
+	// UpgradeChannelNodeImage - Automatically upgrade the node image to the latest version available. Consider using nodeOSUpgradeChannel
```

Comment:
```suggestion
	// UpgradeChannelNodeImage automatically upgrades the node image to the latest version available. Consider using nodeOSUpgradeChannel
```

I think this came up in some of your other PRs but please reformat these doc comments to be a sentence without the dash like in this example. That's the doc convention we're using in the rest of the repo so it would be best to stick with it. It's fine if you want to change that at the end, though, once you've got everything else in this PR ready to go.

Reply:
ack, will keep this in mind for all my upcoming PR's